package.json

```diff
@@ -4,29 +4,33 @@
   "devDependencies": {
     "@types/jasmine": "^3.5.10",
     "clang-format": "^1.2.2",
+    "generate-license-file": "^1.2.0",
     "husky": "^1.3.1",
     "jasmine": "^3.5.0",
+    "prettier": "^2.4.1",
+    "pretty-quick": "^3.1.1",
     "tslint": "^5.9.1",
     "typescript": "^4"
   },
   "engines": {
-    "yarn": ">=1.0",
-    "node": "^12"
+    "node": ">=16.13.0 <17",
+    "npm": ">=8.1.0 <9"
```

Comment:
```suggestion
    "node": "^16.13.0",
    "npm": "^8.1.0"
```

Reply:
I do kinda like the former cuz it's more understandable, but i suppose the `^` appears all over anyway